setup.cfg

```diff
@@ -1,8 +1,42 @@
+[metadata]
+name = soliket
+author = simonsobs
+version = 0.0
```

Comment:
I see `version.py` gets `0.1.dev...` written to it, but I'm not sure where the "0.1" comes from. Should this version here be at least that?